pages/nft/iscn/_iscnId.vue

```diff
@@ -307,12 +308,14 @@ export default class NFTTestMintPage extends Vue {
   }
 
   async checkIsWhitelisted() {
+    logTrackerEvent(this, 'IscnMintNFT', 'CheckIsWhitelisted', this.address, 1);
```

Comment:
I think we should log on the error which is not whitelisted, logging `checkIsWhitelisted` will be spammy since every on submit trigger this now